src/main/java/command/GeneralCommand.java

```diff
@@ -0,0 +1,18 @@
+package command;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.net.http.HttpRequest;
+
+public abstract class GeneralCommand {
+    protected HttpServletRequest req;
+    protected HttpServletResponse resp;
+
+    public void init(HttpServletRequest servletRequest,HttpServletResponse servletResponse){
```

Comment:
Used only on AccountController, probably should be just constructor to not miss setting these fields (you can forget to call _init()_ and req/resp will be null)